app/controller.py

```diff
@@ -66,8 +66,10 @@ def changeToSaveAs(self):
     self.host.changeFocusTo(self.host.interactiveSaveAs)
 
   def doCommand(self, ch):
+    #Check the commandSet for the input with both its string and integer representation.
     self.savedCh = ch
-    cmd = self.commandSet.get(ch)
+    keyName = app.curses_util.cursesKeyName(ch)
```

Comment:
Thanks for working on this. I'm not following a part of this though: what would happen if strings were not used and integers (and tuples) were used? What bug or issue would we see?